src/react/features/diograph/hooks.js

```diff
@@ -1,28 +1,36 @@
 import { useStore } from '../../store'
 
+const useDiory = (id) => {
+  const [{ diograph = {} }] = useStore((state) => state.diograph)
+  return diograph[id]
+}
```

Comment:
I think this as `retrieveDioryFromDiograph` or `getDioryById` . useDiory doesn't feel optimal name for this. But it can also be that I'm not used to it... (and I don't have any better suggestion!)